contracts/lib/MockMessageBus.sol

```diff
@@ -670,7 +641,7 @@ library MockMessageBus {
     {
         messageHash_ =  keccak256(
             abi.encode(
-                _messageTypeHash,
+                // TODO: add type hash
```

Comment:
why is this `todo`?

Reply:
😱 Fixed
However, I had to add a header to the `MessageBus` library. Any better ideas?

/cc @deepesh-kn